pennylane_lightning/lightning_tensor/_tensornet.py

```diff
@@ -196,7 +212,10 @@ def _tensornet_dtype(self):
 
         Returns: the tensor network class
         """
-        return TensorNetC128 if self.dtype == np.complex128 else TensorNetC64
+        if self.method == "tn":
```

Comment:
exact?

Reply:
We use `tn` to follow the naming convenction on `default.tensor`